dspace/modules/api/src/main/java/org/datadryad/rest/storage/rdbms/OrganizationDatabaseStorageImpl.java

```diff
@@ -193,8 +216,24 @@ protected void deleteObject(StoragePath path) throws StorageException {
     }
 
     @Override
-    protected void updateObject(StoragePath path, Organization organization) throws StorageException {
-        throw new StorageException("can't update an organization");
+    protected void updateObject(StoragePath path, DryadJournalConcept journalConcept) throws StorageException {
+        // we need to compare the new journal concept that was created to any matching existing concepts.
+        // then we need to update the original concept with the new one
+        // then we delete the new one.
+        try {
+            Context context = getContext();
+            context.turnOffAuthorisationSystem();
+            String name = journalConcept.getFullName();
+            String status = journalConcept.getStatus();
+            context.commit();
+            DryadJournalConcept conceptToUpdate = JournalUtils.getJournalConceptByJournalName(name);
+            conceptToUpdate.transferFromJournalConcept(context, journalConcept);
+            JournalUtils.removeDryadJournalConcept(context, journalConcept);
+            context.restoreAuthSystemState();
+            completeContext(context);
+        } catch (Exception ex) {
+            throw new StorageException("Exception updating organization: " + ex.getMessage(), ex);
```

Comment:
oh but yeah, I can see that it could have thrown a different exception? I'll add it in anyway.